packages/accordion/src/AccordionItem.ts

```diff
@@ -23,12 +25,19 @@ import chevronIconStyles from '@spectrum-web-components/icon/src/spectrum-icon-c
 
 import styles from './accordion-item.css.js';
 
+const chevronClass = {
+    s: 'spectrum-UIIcon-ChevronRight75',
```

Comment:
```suggestion
    s: () => html`
    <sp-icon-chevron100
            class="indicator spectrum-UIIcon-Chevron75"
        ></sp-icon-chevron100>
    ````

Here's an example of how this would look.